docs/graphql/manual/deployment/downgrading.rst

```diff
@@ -103,6 +103,12 @@ You can downgrade the catalogue from a particular version to its previous versio
   :backlinks: none
   :depth: 1
   :local:
+  
+From 29 to 28
```

Comment:
Catalog version as per this PR is 30

Shouldn't we have 30 to 29 downgrade instructions?

Reply:
Yes, thanks. I forgot to change this when I merged master (which bumped from 29 to 30).